src/Dumplie/Domain/SharedKernel/Money/Currencies.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Domain\SharedKernel\Money;
+
+final class Currencies
```

Comment:
last time when I tried to setup doctirne mapping for external code it didnt end well :P